homeassistant/components/forked_daapd/coordinator.py

```diff
@@ -0,0 +1,142 @@
+"""Support forked_daapd media player."""
+
+from __future__ import annotations
+
+import asyncio
+import logging
+
+from homeassistant.exceptions import PlatformNotReady
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+
+from .const import (
+    SIGNAL_ADD_ZONES,
+    SIGNAL_UPDATE_DATABASE,
+    SIGNAL_UPDATE_MASTER,
+    SIGNAL_UPDATE_OUTPUTS,
+    SIGNAL_UPDATE_PLAYER,
+    SIGNAL_UPDATE_QUEUE,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+WS_NOTIFY_EVENT_TYPES = ["player", "outputs", "volume", "options", "queue", "database"]
+WEBSOCKET_RECONNECT_TIME = 30  # seconds
+
+
+class ForkedDaapdUpdater:
+    """Manage updates for the forked-daapd device."""
```

Comment:
It's not strictly a data update coordinator, but it serves the same function

Reply:
Yeah it is an oddity, but probably still the place one looks for this 👍